src-vue/src/components/PlayButton.vue

```diff
@@ -114,10 +171,24 @@ export default defineComponent({
             />
         </el-option-group>
     </el-select>
+    <el-progress
+        v-if="install_or_update"
+        :format="formatText"
+        :percentage="percentage"
+        :color="color"
+        :indeterminate="status === 'EXTRACTING'"
+        :duration="1"
+    >
+    </el-progress>
 </template>
 
 <style scoped>
 
+/* Set progress bar width */
+.el-progress {
+  width: 40vw;
```

Comment:
`vw` unit is related to your window's width, *i.e.* your progress bar width will increase if you change FlightCore's dimensions.

Reply:
Yeah, this was originally intentional but now did I realise that the rest of the UI doesn't change in size on resizing it's actually kinda stupid to do it like that ^^"

Reply:
Addressed in 9bb4eef1c175dbb07af1fbab283dbdd064914e7f